### ISSUE TITLE
Tab Accessibility violation with header width

### ISSUE THREAD

**Issue by divyajindel** [Sage/carbon#4524]
### Current behaviour

Seeing accessibility warning-
IDs used in ARIA and labels must be unique. Document has multiple elements referenced with ARIA with the same id attribute: tab-1-tab

<img width="1272" alt="Screenshot 2021-10-29 at 14 00 30" src="https://user-images.githubusercontent.com/88449494/139439087-82705dfc-560b-43bd-8a50-e4aa9db01cfe.png">

### Expected behaviour

any Accessibility warning should not be there.

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/tabs--with-header-width

### Carbon Version

Latest

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What Operating System are you seeing the problem on?

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by samtjo** [comment]
FE-4440
